### ISSUE TITLE
Put Downloads in /Releases

### ISSUE THREAD

**Issue by TPS** [svn2github/svn2github#2]
@svn2github @Gabrys Is it possible to replicate the downloads via svn2github in `/releases`, at least for well-known services like SourceForge?  Many projects use other services for binary & other releases, but quite few here on GitHub.  If svn2github were to copy _these_, it'd be eminently useful.

**Comment by gabrys** [comment]
Hi @TPS 

Thank you for the suggestion. svn2github aims to help people convert from SVN to GitHub or to help them access SVN repos while they are more used to working with Git. I don't think the "releases" feature is used that often.

I'm currently working on reincarnating svn2github (the service has not been touched for literally for 4 years now). I want to allow people to contribute some additional information (description, links, etc) so one could list SourceForge links in there.

**Comment by TPS** [comment]
Would this be wiki-style — i.e., folks would be given access to post binaries/links to such themselves — or would the original developers have to do it themselves?  1 of the advantages of svn2gihub as-it-is is that _anyone_ can get anyone _else_'s SVN repo cloned, so being able to post binaries oneself would be equally useful.

**Comment by gabrys** [comment]
Yeah, something wiki-style is in my mind. Probably not uploads, but links -- yes. Actually uploads: yes, if I could hack that, so they are hosted directly on GitHub.

**Comment by TPS** [comment]
Maybe use the wiki system to have the new svn2github script/bot populate `/releases`, somehow, maybe via special syntax, so either cross-load (from external link) or (user) upload (to the wiki) is processed?  That way, everything is kept orderly, & where folks & GitHub expect to find such items.  Also, to pre-process, so that someone doesn't attempt to upload some sort of 287 ZB backup or something.… 😜

**Comment by TPS** [comment]
Another option _might_ be to _optionally_ setup something like https://docs.travis-ci.com/user/deployment/releases, if binaries can't be more simply acquired/uploaded.
